include/internal/common.h

```diff
@@ -175,6 +179,51 @@ __owur static inline int ossl_assert_int(int expr, const char *exprstr,
                            (c)[1]=(unsigned char)(((l)>> 8)&0xff), \
                            (c)[2]=(unsigned char)(((l)    )&0xff)),(c)+=3)
 
+/* NOTE - c is not incremented as per n2l */
+#define n2ln(c,l1,l2,n) { \
+                        c+=n; \
+                        l1=l2=0; \
+                        switch (n) { \
+                        case 8: l2 =((unsigned long)(*(--(c))))    ; \
```

Comment:
This is still using icky old eay style spacing and parentheses.

Reply:
additional commit pushed.  I wonder if these should become `static inline` functions?